wp-includes/functions.php

```diff
@@ -5736,19 +5736,36 @@ function wp_validate_boolean( $var ) {
 /**
  * Delete a file
  *
- * @since 4.2.0
+ * @since 4.9.2
  *
- * @param string $file The path to the file to delete.
+ * @param string $file                The path to the file to delete.
+ * @param array  $directories_unblock Are directory to unblocked. They are ROOT, WP_CONTENT_DIR, WP_ADMIN_DIR
```

Comment:
This is unclear.
Reading the code, these are expected to be the strings `'ROOT'`, `'WP_CONTENT_DIR'` etc, not the constants.
The documentation gives the impression that the constants are expected.